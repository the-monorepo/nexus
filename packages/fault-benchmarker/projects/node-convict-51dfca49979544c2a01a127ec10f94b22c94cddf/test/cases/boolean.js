'use strict';

exports.conf = {
  true: {
    upper: {
      format: Boolean,
      default: null
    },
    lower: {
      format: Boolean,
      default: null
    },
    mixed: {
      format: Boolean,
      default: null
    }
  },
  false: {
    upper: {
      format: Boolean,
      default: null
    },
    lower: {
      format: Boolean,
      default: null
    },
    mixed: {
      format: Boolean,
      default: null
    }
  }
};
