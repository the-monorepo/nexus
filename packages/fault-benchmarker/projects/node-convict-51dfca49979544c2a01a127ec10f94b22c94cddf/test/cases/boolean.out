{
  "true": {
    "upper": true,
    "lower": true,
    "mixed": true
  },
  "false": {
    "upper": false,
    "lower": false,
    "mixed": false
  }
}
